src/main/java/nextstep/favorite/application/dto/FavoriteCreateResponse.java

```diff
@@ -0,0 +1,15 @@
+package nextstep.favorite.application.dto;
+
+import lombok.Builder;
+import lombok.Getter;
+
+@Getter
+public class FavoriteCreateResponse {
+    private Long id;
+
+
+    @Builder
+    public FavoriteCreateResponse(Long id) {
+        this.id = id;
+    }
```

Comment:
불필요한 공백은 제거해 주세요
```suggestion
    @Builder
    public FavoriteCreateResponse(Long id) {
        this.id = id;
    }
```

요구사항의 응답 항목과 일치하도록 FavoriteResponse를 활용해도 괜찮을 것 같은데
FavoriateCreateResponse를 별도로 만들어 주신 이유가 궁금합니다 🤔

Reply:
생성에 대한 Response는 요구사항이 따로 없었던 것 같아서 필요할 것으로 판단되는 id 값만 반환할 수 있도록 수정하였습니다~! 그리고 FavoriteResponse 하나로 등록과 조회 같은 response를 사용하는 방향에 대해서는 어떻게 생각하시는지 궁금합니다!

Reply:
API 스펙상 다른점이 없다면 하나의 response dto를 사용해도 무방하다고 생각합니다
스펙이 달라지는 시점에 분리해도 충분한 것 같아요 😃